### ISSUE TITLE
Script to get the pre-trained weights

### ISSUE THREAD

**Issue by LokLu** [MadryLab/robustness#36]
I cant reproduce the results of standard training as claimed in the repo 95.25%. Is there a specific script can reproduce the result? Thank you!

**Comment by dtsip** [comment]
The files are provided as DropBox urls ending in `dl=0`. You can use `wget` to download these by replacing 0 with 1 (and placing the url in quotes). 

This should download the CIFAR10 standard model:
```wget "https://www.dropbox.com/s/yhpp4yws7sgi6lj/cifar_nat.pt?dl=1"```

**Comment by LokLu** [comment]
@dtsip Thanks for reply, I have downloaded the model. I am more interested in the hyperparameters (training epoch, learning rate) used during the training stage, as I can't reproduce the same performance with my own implementations. I might miss some details there.

**Comment by dtsip** [comment]
Oh, I see. Here are the hyper-parameters we used:

```
arch: resnet50,
epochs: 250,
batch_size: 128,
learning_rate: 0.1,
learning_rate_step: 100, # reduce LR by 10 every 100 epochs
weight_decay: 0.0005,
momentum: 0.9
```

**Comment by LokLu** [comment]
Thanks a lot, I will try again.

**Comment by dtsip** [comment]
Sounds good! Feel free to reopen if this does not solve it.
